core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1724,15 +1724,15 @@ private void prepareError(CairoException ex) {
         int errno = ex.getErrno();
         CharSequence message = ex.getFlyweightMessage();
         prepareErrorResponse(ex.getPosition(), ex.getFlyweightMessage());
-        if (errno == CairoException.NON_CRITICAL) {
-            LOG.error()
-                    .$("error [msg=`").$(message).$('`')
-                    .$(", errno=`").$(errno)
+        if (ex.isCritical()) {
+            LOG.critical()
+                    .$("error [msg=`").$(message)
```

Comment:
message could be in chinese characters, utf8 advised

Reply:
Addressed in 1674eca52172845a7dcc050ae8d77e4ff0ce26a0